.github/workflows/coreweekly.yml

```diff
@@ -0,0 +1,41 @@
+name: Core Weekly
+
+on:
+  schedule:
+    - cron:  '0 7 * * 1'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Build
+      uses: actions/checkout@v2
+    - name: Checkout Core Weekly Generator
+      uses: actions/checkout@master
+      with:
+        repository: PrestaShop/core-weekly-generator
+        path: ./core-weekly-generator
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        cd ./core-weekly-generator
+        pip install -r requirements.txt
+    - name: Generate Core Weekly for the current week
+      run: |
+        cd ./core-weekly-generator
+        python core-weekly.py --week ${date +"%U"} > ../prestashop.github.io/news/_posts/core-weekly/${date +"%Y-%m-%d-coreweekly-%U-%Y.md"}
+    - name: Set Week Num
+      run: echo '::set-output name=WEEK_NUM::${date +"%U of %Y"}'
```

Comment:
It's not really the week number, it's confusing with what you have above `--week ` Maybe you should create two variables, one for WEEK_NUM, one for YEAR, and concatenate both for the title?